buckets/redis/bucket.go

```diff
@@ -90,6 +85,8 @@ func (a *abstractBucket) takeFromRedis(ctx context.Context, client *redis.Client
 	return client.EvalSha(a.factory.scriptSHA, a.keys, args...)
 }
 
+var _ quotaservice.Bucket = (*staticBucket)(nil)
+
 // staticBucket is an implementation of a redisBucket for use with static, named buckets.
```

Comment:
This comment and one on 101 aren't correct now that we've gotten rid of `redisBucket`, right?

Reply:
Ah. True, I didn't even think to look for comments related to `redisBucket` since it seemed basically useless. I'll update them